src/audio/mux/mux.c

```diff
@@ -69,17 +72,52 @@ static int mux_set_values(struct comp_dev *dev, struct comp_data *cd,
 		}
 	}
 
+	/* Verify mixing within one input stream
```

Comment:
not following the comment, we should not be mixing ?

Reply:
Maybe my comment was not clear. Basically, I'm checking here whether user sends to us proper config. If user sends a configurations requiring channels mixing, mux returns an error.